docs/docs/guides/solution-scenarios/device-authorization.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: OAuth 2.0 Device Authorization Flow
+sidebar_label: Device Authorization
+---
+
+ZITADEL implements device authorization as per [RFC 8628](https://datatracker.ietf.org/doc/html/rfc8628). This document demonstrates its use.
+
+## Setup in ZITADEL
+
+1. Go to “Organization”/”Projects”.
+2. Under “Applications” click the “New” button.
+
+    <img src="/docs/img/device-auth/device-auth-01.png" alt="Device Authorization Flow in ZITADEL"/>
+
+3. Select “Native” and enter a name for the application, and click “Continue”.
+
+    <img src="/docs/img/device-auth/device-auth-02.png" alt="Device Authorization Flow in ZITADEL"/>
+
+4. Select “Device Code”. Click “Continue”.
+
+    <img src="/docs/img/device-auth/device-auth-03.png" alt="Device Authorization Flow in ZITADEL"/>
+
+
+5. Check the details and click “Create”.
+
+    <img src="/docs/img/device-auth/device-auth-04.png" alt="Device Authorization Flow in ZITADEL"/>
+
+6. Copy the “Client ID” and store it for later use.
+
+    <img src="/docs/img/device-auth/device-auth-05.png" alt="Device Authorization Flow in ZITADEL"/>
+
+## Device Client Example
+
+The [ZITADEL OpenID Connect client and server library](https://github.com/zitadel/oidc/) written for Go has a device client example, which can behave and authenticate as a device. In order to run this client, you need a recent version of Go (>=1.20) installed on your device.
+
+
+The example requires two environment variables to be set:
+
+
+- `ISSUER`: server address of your instance or domain. You can find the issuer URL in the “URLs” section. In this example, it will be set to [https://test-0o6zvq.zitadel.cloud](https://test-0o6zvq.zitadel.cloud). **Do not use a trailing slash!**
+
+    <img src="/docs/img/device-auth/device-auth-06.png" alt="Device Authorization Flow in ZITADEL"/>
+
+- `CLIENT_ID`: the Client ID we obtained earlier.
+
+
+Replace `ISSUER` and `CLIENT_ID` values with the ones you obtained and run the example as shown below:
+
+
+``` bash
+ISSUER="https://test-0o6zvq.zitadel.cloud" \
+CLIENT_ID="232685602728952637@device_auth" \
+go run github.com/zitadel/oidc/v2/example/client/device@latest
+```
+
+
+You should see some info-level log output with response data and a line with login instructions as given below:
+
+``` bash
+Please browse to https://test-0o6zvq.zitadel.cloud/device and enter code GQWC-FWFK
+INFO[0002] start polling  
+```
+
+At this point, the device app starts polling the token endpoint at 5-second intervals until the request is allowed, denied, or times out (currently 5 minutes).
+
+
+## Authenticate
+When you browse to the given URL and the device code is entered, the authentication flow for a user is started. If you are already logged in, it skips right ahead to the final screen where you can choose to allow or deny the request. If you are not logged in, you will have to enter your credentials for login.
```

Comment:
Markdown requires one newline before and after a header element.

```suggestion
At this point, the device app starts polling the token endpoint at 5-second intervals until the request is allowed, denied, or times out (currently 5 minutes).

## Authenticate

When you browse to the given URL and the device code is entered, the authentication flow for a user is started. If you are already logged in, it skips right ahead to the final screen where you can choose to allow or deny the request. If you are not logged in, you will have to enter your credentials for login.
```

It might work fine for the current renderer, but it is considered [bad practice.](https://www.markdownguide.org/basic-syntax#heading-best-practices)